### DISCUSSION TITLE
feature request: get issue worklog

### DISCUSSION THREAD

**Discussion by segevmatuti1** [ankitpokhrel/jira-cli#493] [Ideas]
Hey, 
I wish I could have - 
`jira issue worklog get ISSUE_KEY`
or 
`jira issue worklog list ISSUE_KEY`
as we may have some worklogs per issue
so I guess it should be presented in a table with --plain option 

happy for your opinion just before I jump into implementing

**Comment by cyberme0w** [comment]
Based on the fact that this has been open for almost two years, am I correct in assuming nobody is working on it? I find being able to view worklogs is rather useful - specially to avoid double entries (did I already add that worklog? Even if it's not in the shell history, I could still have done it in the browser).
